server/src/main/java/org/elasticsearch/cluster/metadata/RolloverConfiguration.java

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+package org.elasticsearch.cluster.metadata;
```

Comment:
Hm, I don't think this should be in `cluster.metadata`.

Would `org.elasticsearch.action.admin.indices.rollover` be a better location? Or potentially a new package under `org.elasticsearch.rollover` ?

Reply:
Ah, good idea, I like the idea to move it to `org.elasticsearch.action.admin.indices.rollover`. I do not think that it qualifies to have its own package. I think it makes sense to bundle it together with the other rollover logic.